packet/models_test.go

```diff
@@ -434,7 +440,8 @@ var cacherTests = map[string]struct {
 			Gateway: net.ParseIP("10.250.142.1"),
 			Netmask: net.ParseIP("255.255.255.0"),
 		},
-		json: discovered,
+		traceparent: "",
```

Comment:
no need to set traceparent to `""` thats what it'll get by default anyway. Setting to default just adds visual noise, I think its best to just have the "set bits" or "things being tested" specified. Can you please remove.

Reply:
I was just trying to be clear but okay.

Reply:
These still need to be dropped

Reply:
Gone in the diff now :)